src/contrib/cluster/Akka.Cluster.Sharding.Tests/Delivery/ReliableDeliveryShardingSpec.cs

```diff
@@ -413,7 +413,7 @@ public async Task
         }
 
         // redeliver also when no more messages are sent to the entity
-        await consumerProbes[1].GracefulStop(RemainingOrDefault);
+        Sys.Stop(consumerProbes[1]); // don't wait for termination
```

Comment:
This might not be the only fix this tests needs, FYI. No way to know for certain other than gathering more data.